src/main/resources/com/ncc/aif/aida_ontology.shacl

```diff
@@ -250,8 +253,11 @@ aida:RelationArgumentShape
   # the object is the entity participating in the event
```

Comment:
the comment is out of date since it only references an entity. Plus it looks like a copy/paste error because it references an event but this is in a RelationArgumentShape